website/blog/2020-01-10-cobalt.md

```diff
@@ -0,0 +1,894 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.0 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+After a long while of silence and with loads of new features we are excited to
+announce the release of Metals v0.8.0, codename `Cobalt`. This release includes
+a huge number of new features and improvements, which we were perfecting over
+the last couple of months.
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+along with contributors from the community. Recently, Metals has also gained
+support from [Lunatech](https://lunatech.com) with the contributions from
+@ckipp01. We are extremely grateful for the growing support from the community
+and businesses alike, which shows in the number of new contributions and their
+quality.
+
+In this release we merged 207 PRs and closed 82 issues, full details:
+https://github.com/scalameta/metals/milestone/9?closed=1
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/) and
+give Metals a try!
+
+- go to implementations
+- rename symbol
+- first code action - import missing symbol
+- support for Pants build tool
+- run and debug for tests and main methods
+- improvements for Bloop integration
+- worksheet support
+- better support for Vim via coc-metals
+- completion to add all abstract members
+
+## Navigation
+
+### Go to implementations
+
+Additional index was created inside of Metals that keeps up-to-date information
+about class hierarchies inside the workspace based on the SemanticDB
+information. This allows us to find both all classes extending the particular
+parent class and symbols being implemented inside of those classes. Due to usage
+of semanticDB, this will work after the workspace is fully compiled.
+
+![implementations](https://i.imgur.com/rke6iny.gif)
+
+Finding implementation also works for symbols outside of the workspace, although
+in that case it might take a while to load the classpath symbol information at
+first. Consecutive calls will reuse cached classpath symbols information, so
```

Comment:
> in that case it might take a while to load the classpath symbol information at
first.
------
in that case, it might take a while to load the classpath symbol information for the first time.

Reply:
Fixed